libraries/core_libs/consensus/include/final_chain/rewards_stats.hpp

```diff
@@ -40,12 +40,12 @@ class RewardsStats {
    */
   bool addTransaction(const trx_hash_t& tx_hash, const addr_t& validator);
 
-  /**
-   * @brief Remove tx from statistics
-   *
-   * @param tx_hash
-   */
-  void removeTransaction(const trx_hash_t& tx_hash);
+  // /**
+  //  * @brief Remove tx from statistics
+  //  *
+  //  * @param tx_hash
+  //  */
+  // void removeTransaction(const trx_hash_t& tx_hash);
```

Comment:
multiple functions are just commented out, why is that ?

Reply:
I see only one and it is used only in code that was previously commented here https://github.com/Taraxa-project/taraxa-node/pull/2209/files#diff-4188af76ed91923bba45dd22c5e3548e24205a7e7872369fd41d5e21b741dc36L108

Reply:
link points to this exact PR ?

Reply:
I think it is some github issue, because this lines are out of changes scope. Link to this in file 
https://github.com/Taraxa-project/taraxa-node/blob/8c2020ad904ead70158828257d1633e1db61eb46/libraries/core_libs/consensus/src/final_chain/rewards_stats.cpp#L114